packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -1,305 +1,14 @@
 import fs from 'fs'
 import path from 'path'
 
-import execa from 'execa'
-import Listr from 'listr'
-import prompts from 'prompts'
 import terminalLink from 'terminal-link'
 
-import { errorTelemetry } from '@redwoodjs/telemetry'
-
-import {
-  getPaths,
-  writeFilesTask,
-  transformTSToJS,
-  getGraphqlPath,
-  graphFunctionDoesExist,
-} from '../../../lib'
-import c from '../../../lib/colors'
-import { isTypeScriptProject } from '../../../lib/project'
-
-const AUTH_PROVIDER_IMPORT = `import { AuthProvider } from '@redwoodjs/auth'`
-
-const OUTPUT_PATHS = {
-  auth: path.join(
-    getPaths().api.lib,
-    isTypeScriptProject() ? 'auth.ts' : 'auth.js'
-  ),
-  function: path.join(
-    getPaths().api.functions,
-    isTypeScriptProject() ? 'auth.ts' : 'auth.js'
-  ),
-}
-
-const WEBAUTHN_SUPPORTED_PROVIDERS = ['dbAuth']
-
-const getWebAppPath = () => getPaths().web.app
-
-const getSupportedProviders = () =>
+export const getSupportedProviders = () =>
```

Comment:
Not looking forward to the merge conflicts I'll have because of this next time I merge from main after this goes in 😬

Reply:
Happy to leave changes to this file out!